### PR TITLE
updated digitizing basic material

### PR DESCRIPTION
**Pull request by Santhosh-M31**

### CODE REVIEW

File: source/docs/3/digitizing_basics.rst [hunk 1]
```diff
@@ -10,7 +10,7 @@ Overview of the task
 We will use a raster topographic map and create several vector layers
 representing features around a park.
 
-Other skills you will learn
+Other skills you will learn.
```
Comment (spatialthoughts) [inline source/docs/3/digitizing_basics.rst:13]: Change to GeoPackage

File: source/docs/3/digitizing_basics.rst [hunk 2]
```diff
@@ -40,287 +40,240 @@ Data Source [LINZ]_
...
+
+ .. image:: /static/3/digitizing_basics/images/07.png
+    :align: center
+   
+08. Now we are ready to start digitizing. We will first create a road layer and digitize the roads around the park area. Select :guilabel:`New GeoPackage Layer...` icon from Panels. A GeoPackage is an open, non-proprietary, platform-independent, and standards-based data format for a geographic information system implemented as an SQLite database container. This makes it much easier to move it around instead of a bunch of shapefiles. In this tutorial, we create a couple of polygon layers and a line layer so that a GeoPackage will be better suited. You can always load a GeoPackage and export layers as a shapefile or any other format you want.
```
Comment (spatialthoughts) [inline source/docs/3/digitizing_basics.rst:78]: Show menu navigation (not toolbar button)

File: source/docs/3/digitizing_basics.rst [hunk 3]
```diff
@@ -40,287 +40,240 @@ Data Source [LINZ]_
...
-    dialog. Choose a thicker line style such as :guilabel:`topo road` from the
-    predefined styles. Click :guilabel:`OK`.
-
-.. image:: /static/3/digitizing_basics/images/13.png
-   :align: center
-
-14. Now you will see the digitized road feature clearly. Click :guilabel:`Save
-    Layer Edits` to commit the new feature to disk.
+   
+13. After you right-click to end the feature, you will get a pop-up dialog called :guilabel:`Road - Feature Attributes`. Here you can enter attributes of the newly created feature. Since the **fid** is an Autogenerate field, you will not be able to enter a value manually. Leave it as such and enter the road name as it appears on the topo map. Optionally, assign a Road Class value as well. Click :guilabel:`OK`.
```
Comment (spatialthoughts) [inline source/docs/3/digitizing_basics.rst:110]: Skip entering any value for fid as it is a sequential id that will be autogenerated

File: source/docs/3/digitizing_basics.rst [hunk 4]
```diff
@@ -40,287 +40,240 @@ Data Source [LINZ]_
...
+
+ .. image:: /static/3/digitizing_basics/images/18.png
+    :align: center 
+	
+19. In the :guilabel:`Snapping options` dialog, click the :guilabel:`Snapping on Intersection`, which allows you to snap on an intersection of a background layer.
+
+ .. image:: /static/3/digitizing_basics/images/19.png
+    :align: center
+   
+20. Now you can click :guilabel:`Add feature` button and digitize other roads around the park. Make sure to click :guilabel:`Save Edits` after adding a new feature to save your work. A helpful tool to help you digitize is the **Vertex Tool**. Click the :guilabel:`Vertex Tool` button and select ``Vertex Tool (Current Layer)``.
```
Comment (spatialthoughts) [inline source/docs/3/digitizing_basics.rst:145]: Highlight Save Edits button in screenshot

File: source/docs/3/digitizing_basics.rst [hunk 5]
```diff
@@ -40,287 +40,240 @@ Data Source [LINZ]_
...
-    :guilabel:`Rectangle from center`.
+ .. image:: /static/3/digitizing_basics/images/24.png
+    :align: center   
+ 
+25. Now select layer ``Parks`` then clickroad :guilabel:`Toggle Editing` and click the :guilabel:`Add feature` button and click on the map canvas to add a polygon vertex. Digitize the polygon representing the park. Make sure you snap to the road's vertices so there are no gaps between the park polygons and road lines. Right-click to finish the polygon.
+
+ .. image:: /static/3/digitizing_basics/images/25.png
+    :align: center
+   
+26. Enter the park name in the :guilabel:`Parks - Feature Attributes` pop-up.
```
Comment (spatialthoughts) [inline source/docs/3/digitizing_basics.rst:183]: Add another step after this showing another polygon drawn.

File: source/docs/3/digitizing_basics.rst [hunk 6]
```diff
@@ -40,287 +40,240 @@ Data Source [LINZ]_
...
+
+ .. image:: /static/3/digitizing_basics/images/31.png
+    :align: center
+   
+32. Right-click to finish the polygon and enter the attributes. Magically the new polygon is shrunk and snapped exactly to the boundary of the neighboring polygons! This is very useful when digitizing complex boundaries where you need not be precise and still have topologically correct polygon. Click :guilabel:`Toggle Editing` to finish editing the ``Parks`` layer.
+
+ .. image:: /static/3/digitizing_basics/images/32.png
+    :align: center
+   
+33.  Now it is time to digitize a building's layer. Create a new polygon layer named ``Buildings`` by clicking on :guilabel:`New GeoPackage Layer...` icon from Panels. Set the :guilabel:`Buildings` and :guilabel:`MuiltiPolygon`. Choose the CRS as ``EPSG:2193 - NZGD 2000``. Click :guilabel:`OK`.
```
Comment (spatialthoughts) [inline source/docs/3/digitizing_basics.rst:219]: buildings

Comment (spatialthoughts) [inline source/docs/3/digitizing_basics.rst:219]: Change to menu navigation

File: source/docs/3/digitizing_basics.rst [hunk 7]
```diff
@@ -40,287 +40,240 @@ Data Source [LINZ]_
...
+
+ .. image:: /static/3/digitizing_basics/images/21.gif
+    :align: center
+   
+22. Once you have finished digitizing all the roads, click the :guilabel:`Toggle Editing` button. Click :guilabel:`Save`.
+
+ .. image:: /static/3/digitizing_basics/images/22.png
+    :align: center
+   
+23. In the :guilabel:`New GeoPackage Layer` dialog, click the :guilabel:`...` button and select the GeoPackage database named ``digitizing.gpkg``. Name the new layer as an attribute called ``Parks``. Select ``MultiPolygon`` as the :guilabel:`Type`. The base topographic map is the ``EPSG:2193 - NZGD 2000`` CRS. Click :guilabel:`OK`. In :guilabel:`New Field` Enter ``Name``, and the :guilabel:`type` as :guilabel:`Text data`, with ``50`` as :guilabel:`Maximum length` and click :guilabel:` Add to Fields List.`. Click :guilabel:`OK`.
```
Comment (spatialthoughts) [inline source/docs/3/digitizing_basics.rst:160]: Now we will create another layer to digitize the parks as polygons.

File: source/docs/3/digitizing_basics.rst [hunk 8]
```diff
@@ -40,287 +40,240 @@ Data Source [LINZ]_
...
+    :align: center
+   
+23. In the :guilabel:`New GeoPackage Layer` dialog, click the :guilabel:`...` button and select the GeoPackage database named ``digitizing.gpkg``. Name the new layer as an attribute called ``Parks``. Select ``MultiPolygon`` as the :guilabel:`Type`. The base topographic map is the ``EPSG:2193 - NZGD 2000`` CRS. Click :guilabel:`OK`. In :guilabel:`New Field` Enter ``Name``, and the :guilabel:`type` as :guilabel:`Text data`, with ``50`` as :guilabel:`Maximum length` and click :guilabel:` Add to Fields List.`. Click :guilabel:`OK`. 
+
+ .. image:: /static/3/digitizing_basics/images/23.png
+    :align: center
+	
 .. note::
 
    **Polygon** - Planar Surface defined by 1 exterior boundary and 0 or more interior boundaries. Each interior boundary defines a hole in the Polygon.
```
Comment (spatialthoughts) [inline source/docs/3/digitizing_basics.rst:167]: Polygon vs. MultiPolygon

**Review by spatialthoughts — CHANGES_REQUESTED** [review]

File: source/docs/3/digitizing_basics.rst (hunk 1 above)
Comment (Santhosh-M31) [reply source/docs/3/digitizing_basics.rst:13]: Done.

File: source/docs/3/digitizing_basics.rst (hunk 2 above)
Comment (Santhosh-M31) [reply source/docs/3/digitizing_basics.rst:78]: Done.

File: source/docs/3/digitizing_basics.rst (hunk 3 above)
Comment (Santhosh-M31) [reply source/docs/3/digitizing_basics.rst:110]: Done.

File: source/docs/3/digitizing_basics.rst (hunk 4 above)
Comment (Santhosh-M31) [reply source/docs/3/digitizing_basics.rst:145]: Done.

File: source/docs/3/digitizing_basics.rst (hunk 5 above)
Comment (Santhosh-M31) [reply source/docs/3/digitizing_basics.rst:183]: Done.

File: source/docs/3/digitizing_basics.rst (hunk 6 above)
Comment (Santhosh-M31) [reply source/docs/3/digitizing_basics.rst:219]: Done.

File: source/docs/3/digitizing_basics.rst (hunk 7 above)
Comment (Santhosh-M31) [reply source/docs/3/digitizing_basics.rst:160]: Done.

File: source/docs/3/digitizing_basics.rst (hunk 8 above)
Comment (Santhosh-M31) [reply source/docs/3/digitizing_basics.rst:167]: Done.

File: source/docs/3/digitizing_basics.rst (hunk 6 above)
Comment (Santhosh-M31) [reply source/docs/3/digitizing_basics.rst:219]: done.

**Pull request merged**